components/build/src/lib.rs

```diff
@@ -173,3 +174,13 @@ where
 pub fn get_exonum_protobuf_files_path() -> String {
     env::var("DEP_EXONUM_PROTOBUF_PROTOS").expect("Failed to get exonum protobuf path")
 }
+
+// The problem is that
+// 1) On Windows path can contain both / and \.
+// 2) `protoc_rust` compares paths as strings but same path can have different representation as a string.
+fn clean_path<P: AsRef<Path>>(path: P) -> PathBuf {
+    path.as_ref()
+        .to_str()
+        .map(|s| PathBuf::from(s.replace("\\", "/")))
```

Comment:
I against such hacks in the code. I'd prefer to create PR into `protobuf-rust` which will fix that issue.